src/Corax/IndexWriter.cs

```diff
@@ -1408,11 +1408,16 @@ private void ResetWriter()
         {
             _pforDecoder.Dispose();
             _indexedEntries.Clear();
+
+            foreach (var entryTerms in _termsPerEntryId.Values)
+                entryTerms.Dispose(_entriesAllocator);
```

Comment:
Shouldn't we just clear this? We are already clearing the allocator, so nothing to be done here?